packages/ibm-products/src/components/Tearsheet/Tearsheet.tsx

```diff
@@ -87,6 +87,14 @@ export interface TearsheetProps extends PropsWithChildren {
    */
   hasCloseIcon?: boolean;
 
+  /**
+   * The content for the header actions area, displayed alongside the title in
+   * the header area of the tearsheet. This is typically a drop-down, or a set
+   * of small buttons, or similar. NB the headerActions is only applicable for
+   * wide tearsheets.
+   */
+  headerActions?: ReactNode;
+
```

Comment:
Hey @wkeese ,
Thanks for the changes. 
I was trying to see this prop type warning in the story `Tearsheet with all header items and influencer`  which passes `headerActions` prop and couldn't find any. Could you help me to reproduce this issue ?

![image](https://github.com/user-attachments/assets/bac3e7a0-924d-4769-9e17-bdadd73e4fcf)

Reply:
@amal-k-joy - I guess `PropTypes` doesn't warn about having **extra** parameters.   I updated the ticket description to just say that we don't have proper `PropTypes` validation, i.e. something like `headerActions={567}` would **not** currently be flagged.

Note that this PR is mainly for fixing the Typescript declaration, or at least that's my main goal.